Testscripts/Windows/GPU-TENSORFLOW.ps1

```diff
@@ -73,22 +76,26 @@ function Main {
             $remoteFiles = "*.csv,*.log"
             Copy-RemoteFiles -download -downloadFrom $allVMData.PublicIP -files $remoteFiles -downloadTo $LogDir `
                 -port $allVMData.SSHPort -username $user -password $password
+            Write-Debug "Collecting remote files like csv or log"
         }
 
         if ($testResult -match "PASS") {
             Write-LogInfo "Generating the performance data for database insertion"
             foreach ($param in $currentTestData.TestParameters.param) {
                 if ($param -match "CUDADriverVersion") {
                     $CUDADriverVersion = $param.Replace("CUDADriverVersion=","").Replace('"',"")
+                    Write-Debug "CUDA driver version: $$CUDADRiverVersion"
```

Comment:
double $ for $$CUDADRiverVersion

Reply:
fixed in 9626be3c614ab3d45d8977198984ea0a43941873